graphene_django/utils/testing.py

```diff
@@ -1,6 +1,6 @@
 import json
 
-from django.test import TestCase, Client
+from django.test import Client, TestCase
```

Comment:
After the change below, is that `Client` class used anywhere?

Reply:
```suggestion
from django.test import TestCase
```